examples/moveit.py

```diff
@@ -65,6 +65,7 @@ def main():
             screen.blit(o.image, o.pos)
 
         pg.display.update()
+        pg.time.delay(100)
```

Comment:
I guess probably a clock would be better than the delay. However, time.delay here is ok too.